### DISCUSSION TITLE
Delete Experiments?

### DISCUSSION THREAD

**Discussion by AMf322qwe** [abrechts/Phoenix-ELN#24] [General]
I created a few experiments to test some things but now I can't get rid of them. Is there a way I can remove these experiments from my view? I know this would create gaps in the experiment numbers but that seems less important to me than the ability to delete experiments.

**Comment by abrechts** [comment]
There are two scenarios to consider:
- Demo experiments: Nothing to worry about here, all demo experiments will be deleted after you create your own username for productive work.
- Non-demo experiments: Instead of deleting an experiment and subsequently creating a new one, the recommended procedure is to replace the content of the experiment to be discarded by new content, which also preserves the continuity of the experiment-ID's. To do this, go to the experiment to be discarded and click the 'Clear' button in the main toolbar. This removes all protocol entries at once, except for the reference reactant. Then replace the remaining reaction sketch by the one of the next experiment you want to create, which also updates the remaining reference reactant properties. Then proceed as usual to build up your next experiment from there.

I hope this helps.

**Reply by AMf322qwe** [reply]
Gotcha thanks for that help.
